src/Map/Sketch/utils.ts

```diff
@@ -1,5 +1,7 @@
 import { useEffect, useRef } from "react";
 
+export const PLUGIN_LAYER_ID_LENGTH = 36;
```

Comment:
What's is this for? This is a little bit tricky 🤔

Reply:
Emmm yes it's tricky, it was for checking whether it's a layer added by layers.add (since its id is from uuid() which has a different length from the id that we used in ReEarth). 
I updated the code, expose a new `isTempLayer` on layers ref to do the check instead of this.